core/src/toga/widgets/imageview.py

```diff
@@ -79,6 +86,8 @@ def __init__(
         # Prime the image attribute
         self._image = None
         self._impl = self.factory.ImageView(interface=self)
+        if PIL_Image!=None and PIL_Image.isImageType(image):
+            image = Image(image) #PIL.Image.Image -> toga.Image
```

Comment:
As I said in my previous review - this isn't needed. `imageview.image = image` calls the setter, and if the argument isn't None or a toga.Image(), a toga.Image is constructed. We can completely defer all type checks to the toga.Image() constructor.